tests/test_kcb.py

```diff
@@ -283,6 +283,63 @@ def test_osbs_build(self, tmpdir, pkg_info, failure, orchestrator):
                 'koji_builds': [koji_build_id]
             }
 
+    def test_private_branch(self, tmpdir):
+        git_uri = 'git://pkgs.example.com/rpms/fedora-docker'
+        git_ref = 'private-test1'
+        source = git_uri + '#' + git_ref
+        src = {'git_uri': git_uri, 'git_ref': git_ref, 'src': source}
+
+        koji_task_id = 123
+        last_event_id = 456
+        koji_build_id = 999
+
+        options = flexmock(allowed_scms='pkgs.example.com:/*:no')
+
+        pkg_info = {'blocked': False}
+        session = flexmock()
+        (session
+            .should_receive('getLastEvent')
+            .and_return({'id': last_event_id}))
```

Comment:
Let's make sure we add `.once()` where necessary in these expectations.

Reply:
I added .times(2) to the callback check.